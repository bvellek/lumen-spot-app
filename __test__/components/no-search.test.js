import React from 'react';
import renderer from 'react-test-renderer';

import NoResults from '../../public/js/components/no-search';

describe('<NoResults />', () => {
  it('it renders correctly', () => {
    const component = renderer.create(
      <NoResults />
    ).toJSON();
    expect(component).toMatchSnapshot();
  });
});
