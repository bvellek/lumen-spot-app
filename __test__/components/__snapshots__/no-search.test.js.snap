// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NoResults /> it renders correctly 1`] = `
<section
  className="no-search"
>
  <p>
    Please use '📍 Current Location' or '🔎 Submit' a desired location in the field above to see results.
  </p>
</section>
`;
